### ISSUE TITLE
Consider using the device admin feature on Android

### ISSUE THREAD

**Issue by evgeniy-ADG** [AdguardTeam/AdguardForAndroid#1341]
It's possible to use SSL filtering without a lock screen using Android administrator privileges. AdClear can do this.

**Comment by Revertron** [comment]
I bet, that it sets a password, adds certificate and then sets empty password. It will not work on every OS version and every phone model, we've tested it before, on my previous job.

**Comment by ameshkov** [comment]
Sounds like a hack, rather ugly one.

**Comment by Revertron** [comment]
Yep, it is a dirty hack. And no way to make it right for user that have set graphical key or pin. It will be messed up :(

**Comment by Evonos** [comment]
they fixed that issue with a non working way ages ago. it should work on any phone how adclear does that. 

and it works great iam atm even using adclear instead of adguard. for the sole reason of everywhere blocking with no issues.
